views/apps.ejs

```diff
@@ -38,7 +38,7 @@
       var result = [];
       for(var i = 0; i <= data.length-1; i++){
         console.log("Printing data "+i+" : "+data[i].imageUrl);
-        result.push(data[i].imageUrl);
+        result.push('<ImagePost imageUrl='+data[i].imageUrl+'/>');
```

Comment:
i think... result.push(<"ImagePost imageUrl={"+data[i].imageUrl+"} />"); this will be the valid think to do while pushing into an array.